src/tzdata/build.jl

```diff
@@ -1,7 +1,7 @@
 # The default tz source files we care about. See "ftp://ftp.iana.org/tz/data/Makefile"
 # "PRIMARY_YDATA" for listing of tz source files to include.
 const STANDARD_REGIONS = [
-    "africa", "antarctica", "asia", "australasia",
+    "africa", "antarctica", "asia", "australasia", "backzone",
```

Comment:
More work needs to be done to determine how we should classify "backzone". From what I have read so far it seems like it should probably be treated like a `LEGACY_REGION`